d2l-localize-mixin.html

```diff
@@ -0,0 +1,18 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="d2l-localize-behavior.html">
+
+<script>
+	window.D2L = window.D2L || {};
+	window.D2L.PolymerMixins = window.D2L.PolymerMixins || {};
+
+	/**
+	* @polymerMixin
+	* @appliesMixin D2L.PolymerBehaviors.LocalizeBehavior
```

Comment:
The polymer linter complains about unrecognized `localize` method in consuming template unless these `@appliesMixin` attributes are present so that the linter can trace everything.